.github/workflows/build.yml

```diff
@@ -305,9 +305,9 @@ jobs:
 
           mkdir -p ./packages/nuget/runtimes/{linux-x64,linux-arm64,osx,win-x86,win-x64}/native
 
-          extract_file ./artifacts/libddwaf-linux-x86_64/libddwaf-*-linux-x86_64*.tar.gz \
+          extract_file ./artifacts/libddwaf-x86_64-linux-musl/libddwaf-*-x86_64-linux-musl*.tar.gz \
             '/libddwaf\.so$' ./packages/nuget/runtimes/linux-x64/native
-          extract_file ./artifacts/libddwaf-linux-aarch64/libddwaf-*-linux-aarch64*.tar.gz \
+          extract_file ./artifacts/libddwaf-aarch64-linux-musl/libddwaf-*-aarch64-linux-musl*.tar.gz \
```

Comment:
There are also 32 bit linux builds (armv7 and x86), are you interested in trying those as well?

Reply:
We don't support 32bits Linux ATM